### PR TITLE
feature/compatility-110

### PR DESCRIPTION
**Pull request by Stolidbug**
upgrade of the plugin in the new architecture

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /node_modules/
 /composer.lock
 
-/etc/build/*
+/etc/build/.gitignore
```
Comment (pocky) [inline .gitignore]: Remove this change

File: composer.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
     },
     "autoload-dev": {
         "classmap": [
-          "tests/Application/Kernel.php"
+            "tests/Application/Kernel.php"
         ]
```
Comment (pocky) [inline composer.json]: Inline this array

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in
 
 ## License and Copyright Attribution
 
-When you open a Pull Request, you agree to license your code under the [EUPL-1.2](LICENSE) and to transfer the copyright on the submitted code to the owner of this project.
+When you open a Pull Request, you agree to license your code under the [EUPL-1.2](../../../../LICENSE) and to transfer the copyright on the submitted code to the owner of this project.
```
Comment (pocky) [inline docs/CONTRIBUTING.md]: Invalid path for LICENSE file

**Review by pocky — CHANGES_REQUESTED** [review]
You need to update your PR according to this one : https://github.com/pocky/SyliusCookieAlertPlugin/pull/3/files

- Remove old files, add new ones.
- PHP files must have a blank line at the end.
- Change the commit message to "Update plugin compatibility to 1.10"

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (Stolidbug) [reply docs/CONTRIBUTING.md]: I had noticed it when I sent my RP it was done afterwards

File: .gitignore (hunk 1 above)
Comment (Stolidbug) [reply .gitignore]: check

File: composer.json (hunk 2 above)
Comment (Stolidbug) [reply composer.json]: check

File: src/EventListener/SlidesUploadListener.php [hunk 4]
```diff
@@ -26,6 +26,7 @@ final class SlidesUploadListener
     public function __construct(SlideUploaderInterface $uploader)
     {
         $this->uploader = $uploader;
+
```
Comment (pocky) [inline src/EventListener/SlidesUploadListener.php:29]: Remove this line

File: src/Repository/BannerRepository.php [hunk 5]
```diff
@@ -19,6 +18,7 @@ public function __construct(ManagerRegistry $registry, string $class)
     {
         $this->manager = $registry->getManagerForClass($class);
         $this->class = $class;
+
```
Comment (pocky) [inline src/Repository/BannerRepository.php:21]: Remove this line

**Review by pocky — CHANGES_REQUESTED** [review]
- Remove old unused files
- Update tests files

File: src/EventListener/SlidesUploadListener.php (hunk 4 above)
Comment (Stolidbug) [reply src/EventListener/SlidesUploadListener.php:29]: check

File: src/Repository/BannerRepository.php (hunk 5 above)
Comment (Stolidbug) [reply src/Repository/BannerRepository.php:21]: check

File: src/EventListener/SlidesUploadListener.php [hunk 6]
```diff
@@ -51,4 +52,4 @@ private function uploadSubjectSlides(BannerInterface $subject): void
             }
         }
     }
-}
+}
```
Comment (pocky) [inline src/EventListener/SlidesUploadListener.php]: Missing EOF

File: src/Repository/BannerRepository.php [hunk 7]
```diff
@@ -38,4 +38,4 @@ public function findBannerForChannel(string $code, ChannelInterface $channel)
 
         return $query->getOneOrNullResult();
     }
-}
+}
```
Comment (pocky) [inline src/Repository/BannerRepository.php]: Missing EOF

File: tests/Application/config/sylius/1.10/bundles.php [hunk 8]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
+    SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
+];
```
Comment (pocky) [inline tests/Application/config/sylius/1.10/bundles.php]: Missing EOF

File: tests/Application/config/sylius/1.10/packages/prod/jms_serializer.yaml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+jms_serializer:
+  visitors:
+    json_serialization:
+      options:
+        - JSON_UNESCAPED_SLASHES
+        - JSON_PRESERVE_ZERO_FRACTION
+    json_deserialization:
+      options:
+        - JSON_UNESCAPED_SLASHES
+        - JSON_PRESERVE_ZERO_FRACTION
```
Comment (pocky) [inline tests/Application/config/sylius/1.10/packages/prod/jms_serializer.yaml]: Missing EOF

File: tests/Application/config/sylius/1.9/bundles.php [hunk 10]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
+    SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
+    FOS\OAuthServerBundle\FOSOAuthServerBundle::class => ['all' => true],
+    Sylius\Bundle\AdminApiBundle\SyliusAdminApiBundle::class => ['all' => true],
+];
```
Comment (pocky) [inline tests/Application/config/sylius/1.9/bundles.php]: Missing EOF

File: tests/Application/config/routes.yaml [hunk 11]
```diff
@@ -1,9 +0,0 @@
-acme_sylius_example_shop:
-    resource: "@AcmeSyliusExamplePlugin/Resources/config/shop_routing.yml"
-    prefix: /{_locale}
-    requirements:
-        _locale: ^[a-z]{2}(?:_[A-Z]{2})?$
-
-acme_sylius_example_admin:
-    resource: "@AcmeSyliusExamplePlugin/Resources/config/admin_routing.yml"
-    prefix: /admin
```
Comment (pocky) [inline tests/Application/config/routes.yaml:9]: You must add your routes (according to the installation guide) like your configuration file in config path.

**Review by pocky — CHANGES_REQUESTED** [review]

**Comment by Stolidbug** [comment]
check

**Pull request merged**